test_elasticsearch_serverless/test_async/test_server/conftest.py

```diff
@@ -37,10 +37,18 @@ async def async_client(elasticsearch_url, elasticsearch_api_key):
     client = None
     try:
         client = elasticsearch_serverless.AsyncElasticsearch(
-            elasticsearch_url, api_key=elasticsearch_api_key, request_timeout=3
+            elasticsearch_url, api_key=elasticsearch_api_key
```

Comment:
Sorry for the unrelated change, but a request timeout of 3 (instead of the default 10) can sometimes be too short on serverless.